package.json

```diff
@@ -4,8 +4,6 @@
     "tsc-check": "tsc --noEmit -p tsconfig.d.json && tsc --noEmit --skipLibCheck",
     "eslint-check": "eslint \"{src,test}/**/*.ts\"",
     "eslint-fix": "eslint \"{src,test}/**/*.ts\" --fix",
-    "prettier-check": "prettier --check .",
-    "prettier-write": "prettier --write .",
```

Comment:
why do we need to delete this?

Reply:
because prettier is now run with `eslint`.